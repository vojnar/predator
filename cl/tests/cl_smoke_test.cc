/*
 * Copyright (C) 2010 Kamil Dudka <kdudka@redhat.com>
 *
 * This file is part of predator.
 *
 * predator is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * any later version.
 *
 * predator is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with predator.  If not, see <http://www.gnu.org/licenses/>.
 */

#include <cl/easy.hh>
#include <iostream>

// required by the gcc plug-in API
extern "C" { int plugin_is_GPL_compatible; }

void clEasyRun(const CodeStorage::Storage &, const char *) {
    std::cout << "cl_smoke_test is up and running..." << std::endl;
}
