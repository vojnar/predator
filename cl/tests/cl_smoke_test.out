cl_smoke_test is up and running...
